src/main/java/modelo/Option.java

```diff
@@ -0,0 +1,24 @@
+package modelo;
+
+public class Option {
+    private Boolean correct;
```

Comment:
en las buenas practicas de todo lenguaje, cuando es un bulean la varaible se le agrega un "is" en el comienzo. 
en este caso me hace un toque de ruido el nombre solo correct